v2/internal/runner/templates.go

```diff
@@ -24,39 +24,29 @@ func (r *Runner) logAvailableTemplate(tplPath string) {
 	}
 }
 
-// listAvailableTemplates prints available templates to stdout
-func (r *Runner) listAvailableTemplates() {
-	if r.templatesConfig == nil {
-		return
-	}
-
-	if _, err := os.Stat(r.templatesConfig.TemplatesDirectory); os.IsNotExist(err) {
-		gologger.Error().Msgf("%s does not exists", r.templatesConfig.TemplatesDirectory)
-		return
-	}
-
+func (r *Runner) listAvailableStoreTemplates(store *loader.Store) {
 	gologger.Print().Msgf(
 		"\nListing available v.%s nuclei templates for %s",
 		r.templatesConfig.TemplateVersion,
 		r.templatesConfig.TemplatesDirectory,
 	)
-	err := filepath.WalkDir(
-		r.templatesConfig.TemplatesDirectory,
-		func(path string, d fs.DirEntry, err error) error {
-			// continue on errors
-			if err != nil {
-				return nil
-			}
-			if d.IsDir() && path != r.templatesConfig.TemplatesDirectory {
-				gologger.Print().Msgf("\n%s:\n\n", r.colorizer.Bold(r.colorizer.BgBrightBlue(d.Name())).String())
-			} else if strings.HasSuffix(path, ".yaml") {
-				r.logAvailableTemplate(path)
-			}
-			return nil
-		},
-	)
-	// directory couldn't be walked
-	if err != nil {
-		gologger.Error().Msgf("Could not find templates in directory '%s': %s\n", r.templatesConfig.TemplatesDirectory, err)
+	extraFlags := r.options.Templates != nil || r.options.Authors != nil ||
```

Comment:
we don't need this extra login i think since we can only display relative path without the other full template display logic. What do you think @ehsandeep